octoprint_mrbeam/util/material_csv_parser.py

```diff
@@ -93,7 +96,7 @@ def parse_csv(path=None, laserhead=MRBEAM):
             if colorcode:
                 current_color = colorcode
 
-            if not mrbeamversion in [MRBEAM, MRB_DREAMCUT, MRB_READY]:
+            if not mrbeamversion in [MRBEAM, MRB_DREAMCUT, MRB_DREAMCUT_S, MRB_READY]:
```

Comment:
Ah yes, I see that it only considers lines with the correct mrbeamversion. I wonder if we could already filter out all of the unrelated mrbeam models here? It's a minor optimisation anyway.

Reply:
will not consider this optimization for now